docker/irods_client/environments/plain-irods/config.toml

```diff
@@ -1,6 +1,7 @@
 password = "rods"
 server_version = "4.3.1"
 can_write_pam_pass = false
+has_pam_pass = true
 resources = ["bundleResc", "demoResc"]
```

Comment:
"bundleResc" is an iRODS 'internal' staging area for handling zip/tar files - it exists to gather/expand those files since their constituent parts could be on different physical resources to begin with.

Reply:
Ah, thank you!!! So far we do not do anything with this resource. Do you have any recommendation for us how approach this resource in the clients (GUI and CLI)? Ignore it, label it with a warning?